contrib/helm/harbor/values.yaml

```diff
@@ -189,22 +189,68 @@ registry:
     pullPolicy: IfNotPresent
   httpSecret: not-a-secure-secret
   logLevel: info
-# comment out one of the below to use your cloud's object storage.
-#  objectStorage:
-#    gcs:
-#      keyfile: ""
-#      bucket: ""
-#      chunksize: "5242880"
-#    s3:
-#      region: ""
-#      accesskey: ""
-#      secretkey: ""
-#      bucket: ""
-#      encrypt: "true"
-#    azure:
-#      accountname: ""
-#      accountkey: ""
-#      container: ""
+  storage:
+    # specify the type of storage: "filesystem", "azure", "gcs", "s3", "swift", 
+    # "oss" and fill the information needed in the corresponding section
+    type: filesystem
```

Comment:
Personally I prefer a more self- explanatory approach, i.e. get rid of `type` and leave only one chunk of storage setting.  
Given the current approach seems also a common practice, I'll approve but let's make sure we keep this consistent